lib/chef/knife/bootstrap.rb

```diff
@@ -175,7 +175,7 @@ def self.friendly_opt_list(opt_array)
       # client.rb content via bootstrap_context
       option :bootstrap_no_proxy,
         long: "--bootstrap-no-proxy [NO_PROXY_URL|NO_PROXY_IP]",
-        description: "Do not proxy locations for the node being bootstrapped; this option is used internally by Chef.",
+        description: "Do not proxy locations for the node being bootstrapped; this option is used internally by #{Chef::Dist::PRODUCT}.",
```

Comment:
@lamont / @btm did we mean Chef the company or Chef the product here?

Reply:
@bobchaos can you make this chef the company

Reply:
@bobchaos Can you just take out everything from the ; on. There's no need to mention the internal nature of this.